### PR TITLE
Use standard installation layout

### LINKED ISSUE #3074: Windows installation layout
Odly, the cmake installation on windows seems to place libraries (dll, static lib, import lib) into

`INSTALLATION_PREFIX/lib/Release` or `INSTALLATION_PREFIX/lib/Debug` (depending on the build type)

instead of

`INSTALLATION_PREFIX/lib/`.

Is this intended?

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixes #3074 as discussed with @XeCycle.

Thanks!

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Ping @XeCycle. I think this can be merged.

**Comment by XeCycle** [comment]
I'm ok with this; I thought this is left to maintainers to decide it.

**Comment by SylvainCorlay** [comment]
Thanks!

FYI that we already include that patch for conda packaging because the `Debug` and `Release` subdirectories are not in the path which is problematic.

File: CMakeLists.txt [hunk 1]
```diff
@@ -584,7 +584,7 @@ if(LIBUV_BUILD_TESTS)
            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-if(UNIX OR MINGW)
+if(UNIX OR MINGW OR MSVC)
```
Comment (vtjnash) [inline CMakeLists.txt:587]: It appears to me this conditional should now just be removed, as this is all platforms, but otherwise SGTM

Comment (SylvainCorlay) [reply CMakeLists.txt]: @vtjnash I updated the PR to remove the conditional as you suggested.

**Review by vtjnash — APPROVED** [review]
CI: https://ci.nodejs.org/job/libuv-test-commit/2168/

But might be good to get at least one other reviewer too

**Comment by vtjnash** [comment]
CI seems fine with this. Anyone else want to review also?

**Pull request merged**